frontend/talentsearch/src/js/api/poolAdvertisementOperations.graphql

```diff
@@ -0,0 +1,47 @@
+mutation createPoolAdvertisement(
+  $userId: ID!
+  $poolAdvertisement: CreatePoolAdvertisementInput!
+) {
+  createPoolAdvertisement(userId: $userId, poolAdvertisement: $poolAdvertisement) {
+    id
+  }
+}
+
+mutation updatePoolAdvertisement(
+  $id: ID!
+  $poolAdvertisement: UpdatePoolAdvertisementInput!
+) {
+  updatePoolAdvertisement(id: $id, poolAdvertisement: $poolAdvertisement) {
+    id
+  }
+}
+
+mutation publishPoolAdvertisement(
+  $id: ID!
+  $poolAdvertisement: PublishPoolAdvertisementInput!
```

Comment:
What is the purpose of the $poolAdvertisement parameter?  Wouldn't you just need the ID for this action?

Reply:
Don't I want to validate the arguments that get passed in?